src/test/java/org/minions/devfund/mauricioz/linkedList/InsertSortedTest.java

```diff
@@ -0,0 +1,181 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+// Source code recreated from a .class file by IntelliJ IDEA.
+// (powered by Fernflower decompiler)
+*/
+public class InsertSortedTest {
+    private static final int LISTFIRSTVALUE = 3;
+    private static final int LISTSECONDVALUE = 8;
+    private static final int LISTTHIRDVALUE = 11;
+
+    private static final int LISTSIZE1 = 4;
+    private static final int LISTSIZE2 = 3;
+
+    private static final int LISTNEWVALUE1 = 10;
+    private static final int LISTNEWVALUE2 = 14;
+    private static final int LISTNEWVALUE3 = 1;
+
+    private static final int LISTPOSITION3 = 3;
+    /**
+     * constructor.
+     */
+    public InsertSortedTest() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)